### ISSUE TITLE
When scaffolding the HasCharSet() method won't compile

### ISSUE THREAD

**Issue by Slushnas** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#911]
### Steps to reproduce
Scaffold a table with a string column e.g. varchar(128).

### The issue
Scaffolding produces code:
```
entity.Property(e => e.Columnname)
                    .IsRequired()
                    .HasColumnName("columnname")
                    .HasColumnType("varchar(128)")
                    .HasCharSet("utf8mb4");
```
The HasCharSet() method has the following compilation error:
` Argument type 'string' is not assignable to parameter type 'Pomelo.EntityFrameworkCore.MySql.Storage.Internal.CharSet'`

### Further technical details

MySQL version: **10.1.41-MariaDB-0ubuntu0.18.04.1**
Operating system: **Ubuntu 18.04**
Pomelo.EntityFrameworkCore.MySql version: **3.0.0-rc2.final**
Microsoft.AspNetCore.App version: **3.0**

**Comment by lauxjpn** [comment]
> Argument type 'string' is not assignable to parameter type 'Pomelo.EntityFrameworkCore.MySql.Storage.Internal.CharSet'

That's compiler speak for "*use a `CharSet` object instead of a `string`, dude*".
The following code should work:

```c#
entity.Property(e => e.Columnname)
    .HasCharSet(CharSet.Utf8Mb4);
```

I will check, why the scaffolder returns a string instead of a `CharSet.Utf8Mb4`.

**Comment by mguinness** [comment]
Just an observation, but reading [Column Character Set and Collation](https://dev.mysql.com/doc/refman/8.0/en/charset-column.html) I wonder if would be preferable to omit `HasCharSet()` when the column charset matches the table.

**Comment by lauxjpn** [comment]
@mguinness That is definitely something we should consider in the future. We would have to annotate the table with a charset as well to support that.
Please open an issue for that, so we don't forget it!

**Comment by lauxjpn** [comment]
@Slushnas Thank you for reporting this issue! It is fixed with #914.
